src/sentry/tasks/post_process.py

```diff
@@ -937,8 +937,12 @@ def process_workflow_engine(job: PostProcessJob) -> None:
     if job["is_reprocessed"]:
         return
 
-    # TODO - Add a rollout flag check here, if it's not enabled, call process_rules
-    # If the flag is enabled, use the code below
+    # TODO: only fire one system. to test, fire from both systems and observe metrics
+    if not features.has(
+        "organizations:workflow-engine-issue-alert-rollout", job["event"].project.organization
+    ):
+        return
```

Comment:
@snigdhas are you still testing metric issues? this flag will need to be turned on to have post process run but it shouldn't block creating the issue